### ISSUE TITLE
Installation failed (osx-arm64 hardware architecture)

### ISSUE THREAD

**Issue by sarahsurrain** [orasanen/ALICE#29]
When installing on macOS Sonoma 14.5, I get the errors in this screenshot:
![Screenshot 2024-07-06 at 12 02 11 PM](https://github.com/orasanen/ALICE/assets/57930051/80d94f76-ee92-4111-9305-99abc1c66e10)

**Comment by MarvinLvn** [comment]
Hi there,

Thanks for posting the issue!
Unfortunately, I cannot reproduce your error, which I think is specific to your hardware architecture, `osx-arm64`. 
Some packages are never released on this architecture, hence the error you got.

**1st idea**

Could you try running these commands from within the ALICE folder:

```
CONDA_SUBDIR=osx-64 conda env create -f ALICE_macOS.yml
conda activate ALICE
conda config --env --set subdir osx-64
```
 
 and let me know what you get? (no need to run the two last commands if the first one fails)
 
 **2nd idea**
 
 Alternatively, if the 1st idea doesn't work, can you try:
 
 ```
 conda env create -f ALICE_Linux.yml 
 ```
 
 The ALICE_Linux.yml seems more flexible than the MAC-specific .yml file. 
 If you're lucky, this might be sufficient to solve your problem. But I don't have much hope since the error seems to arise with very common packages.
 
 In any case, please let me know what you get with the 1st idea!
 
 P.S.: some users reporting similar errors [here](https://stackoverflow.com/questions/70205633/cannot-install-python-3-7-on-osx-arm64) or [there](https://stackoverflow.com/questions/71515117/how-to-set-up-a-conda-osx-64-environment-on-arm-mac)

**Comment by sarahsurrain** [comment]
Thanks! I was able to get it working using the 1st idea above. In my case, there seemed to be a few problems. One was that my internet connection was slow so a few processes timed out. When that happened, re-running the commands didn't work. I had to delete all of the created files and directories and the conda environment and start from scratch. Also, after I got the installation to work, my test file seemed to be a problem. When I renamed it "test.wav" and moved it to my Desktop, it worked fine. The original file in its original location caused an error. I still don't know if it was special characters in the filename, the path length, or the fact that it was in a cloud-synced directory that was the problem. Thanks again!
